ui/models/job.js

```diff
@@ -83,7 +83,7 @@ export default class JobModel {
     return JobModel.getList(options, config);
   }
 
-  static async retrigger(jobs, repoName, notify, times = 1) {
+  static async retrigger(jobs, repoName, notify, currentRepo, times = 1) {
```

Comment:
Clearly I have some code cleanup to do here...  :)  ``repoName`` is never used.  But you're right to just leave it here.  I'll take care of it in a follow-up PR.

Reply:
Don't apologize, it's not bad :)  The cleanup was because I had *added* a new parameter that made the old one redundant.